robocorp-code/vscode-client/src/viewsLocators.ts

```diff
@@ -84,6 +85,8 @@ export class LocatorsTreeDataProvider
             iconPath = "error";
         } else if (type === "windows") {
             iconPath = "multiple-windows";
+        } else if (type === "web-recorder") {
```

Comment:
Since the `InspectorType` enum is now available, I guess that this should be `InspectorType.WebRecorder` (and the other related `ifs` should be updated too...).